src/app/components/search-bar/search-bar.component.html

```
@@ -1,18 +1,18 @@
-<div>
+<div class="toolbar-placeholder">
   <mat-toolbar>
     <div class="content">
-      <button mat-flat-button aria-label="back" class="button backButton" (click)="goBack()">
+      <button mat-flat-button aria-label="back" class="button actionButton backButton" (click)="goBack()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)